readme.md

```diff
@@ -15,18 +15,61 @@ Control the Neopixels directly attached to the board.
 ```js
 var Playground = require("playground-io");
 var five = require("johnny-five");
-var board = new five.Board({
-  io: new Playground({
-    port: "/dev/tty.usbmodem1411"
-  })
+var io = new Playground({
+  port: "/dev/tty.usbmodem1411"
 });
+var board = new five.Board({ io: io });
 board.on("ready", function() {
   var pixels = Array.from({ length: 10 }, function(_, index) {
     return new five.Led.RGB({
       controller: Playground.Pixel,
       pin: index
     });
-  });
+  }); 
+  
+  var led = new five.Led(13); 
+  
+  var buttonL = new five.Button('4', {
+    isPullup: true,
+    invert: true
+  }); 
```

Comment:
This code isn't valid ;)

It should be: 

```js
var buttonL = new five.Button({
  pin: 4
  isPullup: true,
  invert: true
}); 
```

However, `isPullup: true` and `invert: true` will effectively do the same thing in most cases.